src/Compilers/CSharp/Portable/Utilities/FunctionExtensions.cs

```diff
@@ -12,9 +13,35 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal static partial class FunctionExtensions
     {
+#if (NETCOREAPP)
```

Comment:
i don't understand why we need both of these functions.

Reply:
Extension method `ToHash<T>` is not available in `.netstandard2.0`.